server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds.Contracts/ApplicationTypeEnum.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Arcadia.Assistant.AppCenterBuilds.Contracts
+{
+    public enum ApplicationTypeEnum
```

Comment:
what are you, hungarian? let's rename all the classes to be MyDataClass : IMyDataInterface

Reply:
Done